scene/gui/text_edit.cpp

```diff
@@ -1255,7 +1270,8 @@ void TextEdit::_notification(int p_what) {
 						// Draw fold markers.
 						if (draw_fold_gutter) {
 							int horizontal_gap = (cache.fold_gutter_width * 30) / 100;
-							int gutter_left = cache.style_normal->get_margin(MARGIN_LEFT) + cache.breakpoint_gutter_width + cache.line_number_w + cache.info_gutter_width;
+							int gutter_left = cache.style_normal->get_margin(MARGIN_LEFT) + cache.breakpoint_gutter_width + cache.line_number_w + cache.info_gutter_width + 2 * cache.gutter_indicator_width;
```

Comment:
@Paulb23 Could you help me to set margin and space between line no and fold gutters so line change gutter do not overlap on fold gutters?  This is how it looks currently
![Screenshot from 2020-08-07 11-17-30](https://user-images.githubusercontent.com/29702428/89613751-9981a980-d8a0-11ea-9ed4-ea150265a00f.png).
I need 2px space between line no and fold gutters for line change gutters.

Reply:
The line number gutter pads by an additional char, the fold gutter currently draws over this padding, so the fold icon draw code needs updating.

However, if possible I would recommend holding on for #40973 first, where this is fixed and in general should make gutters easier.

Reply:
@Paulb23 I am currently working the 3.2 branch and I will later port this to master.